src/routes/docs/tutorials/react/step-4/+page.markdoc

```diff
@@ -29,6 +30,7 @@ export function UserProvider(props) {
   async function login(email, password) {
     const loggedIn = await account.createEmailPasswordSession(email, password);
     setUser(loggedIn);
+    goTo("/");
```

Comment:
As discussed on our daily, this won't work in pure react. Please look into using window.location, but also include a comment to let the developer know they should use the appropriate method for their scenario.